core/config/src/main/scala/edu/uci/ics/texera/config/DefaultsConfig.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package edu.uci.ics.texera.config
+
+import com.typesafe.config.{ConfigFactory, ConfigRenderOptions, ConfigValueType}
+import edu.uci.ics.amber.util.PathUtils
+import java.nio.file.Files
+import java.util.Base64
+import scala.jdk.CollectionConverters.MapHasAsScala
+
+object DefaultsConfig {
+  private val conf = ConfigFactory.parseResources("defaults.conf").resolve()
+  private val imageKeys = Set("logo", "mini_logo", "favicon")
```

Comment:
This function is not necessary. We don't need to convert this path to Base64 here. We can either do the conversion offline and save the Base64 value or use the path directly in the defaults.conf. Since using a path is easier, we can use a path.